skyve-war/src/main/java/modules/admin/Configuration/Configuration.xml

```diff
@@ -43,6 +43,12 @@
 			<displayName>admin.configuration.passwordRuleDescription</displayName>
 			<length>250</length>
 		</text>
+		<boolean name="haveIBeenPwnedEnabled">
+			<documentation>Determines whether or not HaveIBeenPwned API is used as part of password validation.</documentation>
+			<displayName>HaveIBeenPwned API enabled</displayName>
+			<description>The HaveIBeenPwned API checks whether a password has been compromised in known data breaches.</description>
```

Comment:
I wonder if we should change the terminology for the end user to be more generic, as it is unlikely they know what HaveIBeenPwned is.

Something along the lines of:

Check for breached password
When users try to create or change a password, checks whether the new password has been compromised in known data breaches (requires internet access).